invoker/core/src/main/java/com/google/cloud/functions/invoker/http/TimeoutFilter.java

```diff
@@ -49,13 +47,13 @@ public void run() {
                 ((HttpServletResponse) response)
                     .sendError(HttpServletResponse.SC_REQUEST_TIMEOUT, "Request timed out");
               } catch (IOException e) {
-                e.printStackTrace();
+                logger.warning("Error while sending HTTP response " + e.toString());
```

Comment:
Nit: do we want a colon after the response to differentiate the exception from the message text? Also on line 56.